azkaban-exec-server/src/main/java/azkaban/execapp/JobRunner.java

```diff
@@ -770,7 +770,6 @@ private boolean prepareJob() throws RuntimeException {
 
       insertJobMetadata();
       insertJVMAargs();
-
```

Comment:
extreme nit : restore this blank line, it maintains readability.

Reply:
Done